apps/marketplace/components/BuyerATM/BuyerATMAdditionalInformationStage.js

```diff
@@ -2,21 +2,53 @@ import React, { Component } from 'react'
 import PropTypes from 'prop-types'
 import { connect } from 'react-redux'
 import { actions, Form } from 'react-redux-form'
-import { required, validPhoneNumber, dateIs2DaysInFuture } from 'marketplace/components/validators'
 import formProps from 'shared/form/formPropsSelector'
 import Textfield from 'shared/form/Textfield'
-import AUheading from '@gov.au/headings/lib/js/react.js'
+import AUheadings from '@gov.au/headings/lib/js/react.js'
```

Comment:
Is there a reason why this import is changing in this pull request?

Reply:
I am keeping it consistent with the other flows; Specialist, Training etc. Most places in the code it refers to as AUHeadings.

Reply:
We don't have consistent usage across the front-end but there is greater use of `AUheading` which the Design System references in their documentation.  If we're worried about consistency, I'd do it in a separate pull request.

Reply:
Fixed - I created a separated PR and fixed it in the frontend repo.